crates/language/src/php_only.rs

```diff
@@ -0,0 +1,118 @@
+use std::sync::OnceLock;
+use lazy_static::lazy_static;
+use regex::Regex;
+
+
+use crate::language::{fields_for_nodes, Field, Language, SortId, TSLanguage};
+
+static NODE_TYPES_STRING: &str = include_str!("../../../resources/node-types/php_only-node-types.json");
+
+static NODE_TYPES: OnceLock<Vec<Vec<Field>>> = OnceLock::new();
+static LANGUAGE: OnceLock<TSLanguage> = OnceLock::new();
+
+#[cfg(not(feature = "builtin-parser"))]
+fn language() -> TSLanguage {
+    unimplemented!(
+        "tree-sitter parser must be initialized before use when [builtin-parser] is off."
+    )
+}
+#[cfg(feature = "builtin-parser")]
+fn language() -> TSLanguage {
+    tree_sitter_php::language_php_only().into()
+}
+
+#[derive(Debug, Clone)]
+pub struct PhpOnly {
+    node_types: &'static [Vec<Field>],
+    metavariable_sort: SortId,
+    language: &'static TSLanguage,
+}
+// use std::io::Write;
+impl PhpOnly {
```

Comment:
We should follow the pattern from JS of having some shared x-script utils that get used for both flavors instead of repeating everything.